clients/web/src/views/layout/ProgressListView.js

```diff
@@ -51,6 +54,15 @@ var ProgressListView = View.extend({
         this._onUpdate();
     },
 
+    _clearProgress: function () {
+        _.each(Object.keys(this._map), (key) => {
```

Comment:
This could be simplified (and use clearer variable names) as 
```
_.each(this._map, (progressId, progressWidget) => {
    progressWidget.destroy();
    ...
```